<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGame</class>
 <widget class="QDialog" name="NewGame">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,3,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="title">
         <string>White</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QRadioButton" name="white_human">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Human</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="white_computer">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Computer</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="title">
         <string>Black</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QRadioButton" name="black_human">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Human</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="black_computer">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Computer</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="title">
         <string>Rules</string>
        </property>
        <widget class="QCheckBox" name="mosquito_check">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Mosquito</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="ladybug_check">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Ladybug</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="pillbug_check">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Pillbug</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="queen_first_check">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Cannot place
queen first</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="title">
         <string>Computer strength</string>
        </property>
        <widget class="QRadioButton" name="easy">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>93</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Easy</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="medium">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>93</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Medium</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="hard">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>93</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Hard</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="expert">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>93</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Expert</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
